src/google-maps/google-map/google-map.ts

```diff
@@ -399,7 +399,7 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
    */
   getMapTypeId(): google.maps.MapTypeId|string {
     this._assertInitialized();
-    return this.googleMap.getMapTypeId();
+    return this.googleMap.getMapTypeId()!;
```

Comment:
I'm not a fan of these non-null assertions, but the new type is `| undefined` which is a breaking change.

Reply:
Do we do anything in our wrapper components that ensure that these values are defined? If not, we should probably just accept the breaking change for v13. The maps stuff has very low usage inside Google, so I don't imagine it would be difficult to land.

Reply:
Done.